src/components/ContinuationIn/ExtraproBusinessLookup.vue

```diff
@@ -0,0 +1,172 @@
+<template>
+  <div id="extrapro-business-lookup">
+    <div class="input-block">
+      <v-autocomplete
+        v-model="selectedBusiness"
+        filled
+        no-filter
+        append-icon=""
+        return-object
+        autocomplete="chrome-off"
+        menu-props="{ maxHeight: 380 }"
+        hint="Enter at least 3 characters"
+        hide-details="auto"
+        item-text="identifier"
+        :disabled="state === States.SUMMARY"
+        :label="businessLookupLabel"
+        :name="Math.random()"
+        :items="searchResults"
+        :loading="state == States.SEARCHING"
+        :hide-no-data="state != States.NO_RESULTS"
+        @update:search-input="onSearchInput($event)"
+      >
+        <!-- Empty selection slot will stop re-triggering of search-input event -->
+        <template #selection="" />
+
+        <template #no-data>
+          <v-list-item>
+            <div>No matches found.</div>
+          </v-list-item>
+        </template>
+
+        <template #item="{ item }">
+          <v-row class="business-lookup-result pt-1">
+            <v-col
+              cols="3"
+              class="result-identifier d-inline-flex"
+            >
+              {{ item.identifier }}
+              <!-- *** TODO: remove after testing: -->
+              ({{ item.status?.charAt(0) }})
+            </v-col>
+
+            <v-col
+              cols="7"
+              class="result-name flex-1-1"
+            >
+              {{ item.name }}
+            </v-col>
+
+            <v-col
+              cols="2"
+              class="result-action"
+            >
+              Select
+            </v-col>
+          </v-row>
+        </template>
+      </v-autocomplete>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component, Emit, Watch } from 'vue-property-decorator'
+import { debounce } from 'lodash'
+import { BusinessLookupResultIF } from '@/interfaces'
+import { BusinessLookupServices } from '@/services'
+
+/* eslint-disable no-unused-vars */
+enum States {
+  INITIAL = 'initial',
+  SEARCHING = 'searching',
+  SHOW_RESULTS = 'show results',
+  NO_RESULTS = 'no results',
+  SUMMARY = 'summary'
+}
+/* eslint-enable no-unused-vars */
+
+//
+// This component is a hybrid between the shared BusinessLookup component
+// and the Auth Web BusinessLookup component.
+//
```

Comment:
Unfortunately, I couldn't use the shared component here (since it also handles the summary display and has hard-coded values).

Also unfortunately, I found a local copy of this component in Auth Web. However, we might be able to reuse this one...

Some time in the future, a good plan might be to break up the shared component into a search-only sub-component, and a wrapper that would also handle the summary display. I think here, and in Auth Web, we could use just the search-only component (and do our own display). Meanwhile, other users of the shared component could use the wrapper that does both. In this way we would have maximum flexibility and reduced code duplication.